docs/home/howto/BACKUP.md

```diff
@@ -12,12 +12,12 @@ Epiphany provides solution to create full or partial backup and restore for some
 - [Kubernetes (only backup)](#kubernetes)
 
 Backup is created directly on the machine where component is running, and it is moved to the ``repository`` host via
-rsync. On the ``repository`` host backup files are stored in location ``/epibackup/mounted`` mounted on a local
+rsync. On the ``repository`` host backup files by default are stored in location ``/epibackup/mounted`` mounted on a local
```

Comment:
Is this location configurable?
I see it in ansible/playbooks/roles/backup/defaults/main.yml and not in schema/common/defaults/configuration/backup.yml